app/src/main/res/values/strings.xml

```diff
@@ -1791,6 +1791,9 @@
     <!-- Default browser experiment -->
     <string name="default_browser_experiment_card_text">Set links from websites, emails, and messages to open automatically in Firefox.</string>
 
+    <!-- Default browser experiment -->
+    <string name="default_browser_experiment_card_title">Title</string>
```

Comment:
Since this is only used in the @preview, can we avoid adding a string that would be localized.